Import/index.mjs

```diff
@@ -0,0 +1,77 @@
+import {Manifest} from "manifesto.js"
+import dbDriver from "../database/driver.mjs"
+
+const database = new dbDriver("mongo")
+
+export class Project {
+  constructor(data) {
+    this.data = data
+  }
+
+  async save() {
+    return database
+      .save({...this.data, "@type": "Project"})
+      .then((savedProject) => {
+       
+        return savedProject
+      }) 
+  }
+
+  static async fetchManifest(projectId) {
+    const url = `https://t-pen.org/TPEN/project/${projectId}`
+    return fetch(url)
+      .then((response) => {
+        return response.json()
+      })
+   }
+
+  static async processManifest(projectId) { 
+    return Project.fetchManifest(projectId).then((manifest) => { 
+       // Process manifest into new Project
+      //Return newProject
+      // UNDER CONSTRUCTION
+
+      let newProject = {}
+
+      const iiifManifest = new Manifest(manifest["@id"]) 
+      const sequences = iiifManifest.getSequences()
+
+      newProject.sequences = sequences
+      newProject.metadata = iiifManifest.getMetadata()
+      newProject.label = iiifManifest.getLabel()
+
+    
+      return newProject
+    }).catch((err)=>{
+      console.log(err)
+    })
+
+
+  }
+
+  static async importProject(projectId) {
+    return Project.fetchManifest(projectId)
+      .then((manifest) => { 
+        return Project.processManifest(manifest)
+      })
+      .then((newManifest) => {
+        const project = new Project(newManifest)
```

Comment:
processManifest says `Process manifest into new Project`.

What you have in this `then()` is a new TPEN3 Project, not a Manifest.  Isn't new Project(project) redundant?  Or will processManifest(manifest) return a Manifest?

Remember that Project !== Manifest

A Project has Pages, Layers, Lines that it can assemble and return as a Manifest.